### ISSUE TITLE
Text / usability

### ISSUE THREAD

**Issue by philip-n** [inbaz/ers#192]
I know the system by now and have an understanding of what works how and which components tie into each other. As some parts may not be obvious immediately, I'd try to find some time on wednesday evening and come up with some text enhancements (or help texts) to guide the user.

@inbaz, @smadamit, @jackcaveman: If someone else is already doing this, let me know and I'll pass. Also, are there plans for translations (sooner or later)?

@inbaz: What would be the easiest way to provide you with text change proposals for a couple of different sites?

**Comment by inbaz** [comment]
@philip-n Please send this improvements via mail with the according link at the top and the test below. Perfect would be when old text is striked through and new text is colored, i.e. red.

A translation module is planned - later, not sooner. :)

**Comment by smadamit** [comment]
@philip-n I will do no text testing, so would be nice if you do it! I concentrate my testing on the functionality.

If there is text which depends on the configuration in the admin panel I have to know the change requests. I will do these changes. Andi will do text changes in the code.

The prereg will start in English without translations.

**Comment by smadamit** [comment]
see also: #209

**Comment by smadamit** [comment]
see also: #204

**Comment by smadamit** [comment]
see also: #215

**Comment by smadamit** [comment]
all profile features will be implemented after going live. I think we should avoid all link like register, login, etc. in the text or we leave them, but route to a page with the warning: "not implemented yet"

example: register link of the home site

**Comment by philip-n** [comment]
Concerning last comment: I'd rather leave out hint to profile completely. I wouldn't feel good when I'm shopping on a site that tells me it is not completely implemented yet - who knows what else might be missing there? ;-)

**Comment by philip-n** [comment]
Okay finished (or rather: stopped) on this for today. I've typed eveything into a .odt document and mailed it to you.

**Comment by inbaz** [comment]
I went through the list and did most of the changes. Please have a look.

**Comment by philip-n** [comment]
Just had a quick look, most looks good! Will scrutinize later ;-)

One thing about the product mask as of now:
![03 ticket selection new](https://cloud.githubusercontent.com/assets/3903619/6769764/45813e00-d0a8-11e4-9008-86286286242e.jpg)
To me, the informations are quite spread out and difficult to grasp. There is one block with the description, one with the prices and button (which is not accessible), one with a yellow box, one with the gala-ticket-inclusion, and one with a drop down.

I'd prefer a linear layout where I can read from top to bottom, do the things I need to do along the way (such as adding a person), and when I arrive at the bottom, I click the (then activated) "Add to cart". See mockup below.
![03 ticket selection proposed](https://cloud.githubusercontent.com/assets/3903619/6769765/458170aa-d0a8-11e4-84c8-b1be4e137548.jpg)

@inbaz @ocirne @smadamit @jackcaveman : Comments? Pro, contra?

**Comment by smadamit** [comment]
I do absolutely agree to philip-n -> new layout

**Comment by christianbross** [comment]
@philip-n 
we want to move this out of the current critical go-live timeline - may be many topics are solved already
if you don't have anything where you feel really bad, I'll move it out of the go-live milestone - otherwise please let us know what is still critical/important for usability for go-live

**Comment by philip-n** [comment]
@christianbross : I totally agree. Text changes are implemented as far as I can tell, layout is rather to be discusse dthan urgent.
